test/requirements.txt

```diff
@@ -2,10 +2,10 @@ docker==4.2.0
 fabric==2.5.0
 junit-xml==1.9
 packaging==20.1
-pytest==5.3.5
-pytest-rerunfailures==9.0
+pytest==6.2.3
+pytest-rerunfailures==9.1.1
 pytest-timeout
-pytest-xdist==1.31.0
+pytest-xdist==2.2.1
```

Comment:
Do we need to upgrade these versions to make use of some functionality that isn't available right now, or is it more of a general currency update?

(Either way, thanks.)

Reply:
There is an inconsistency in functionality of pytest when `rerun` and `-n=auto` options are used in a single command for older version of `pytest` and related plugins. 
Upgrading the same to the latest version resolves the issue.